components/automate-cli/cmd/chef-automate/status.go

```diff
@@ -47,6 +75,22 @@ func newStatusCmd() *cobra.Command {
 
 	return statusCmd
 }
+func newStatusSummaryCmd() *cobra.Command {
+	var statusSummaryCmdFlags = StatusSummaryCmdFlags{}
+	var statusSummaryCmd = &cobra.Command{
+		Use:   "summary",
+		Short: "Retrieve Chef Automate status-summary",
+		Long:  "Retrieve Chef Automate status node summary for HA deployment",
+		RunE:  runStatusSummaryCmdFunc(&statusSummaryCmdFlags),
+	}
+	statusSummaryCmd.PersistentFlags().BoolVarP(&statusSummaryCmdFlags.isAutomate, "automate", "a", false, "Get only automate Status")
+	statusSummaryCmd.PersistentFlags().BoolVarP(&statusSummaryCmdFlags.isChefServer, "chef-server", "c", false, "Get only chef server Status")
+	statusSummaryCmd.PersistentFlags().BoolVarP(&statusSummaryCmdFlags.isOpenSearch, "opensearch", "o", false, "Get only opensearch Status")
```

Comment:
add short flags for chef-server as cs, opensearch as os, and PostgreSQL as pg

Reply:
getting pic error after doing changes (chef-server as cs, opensearch as os, and PostgreSQL as pg)